src/modules/Providers/views/list/components/StatusCell.tsx

```diff
@@ -12,6 +12,26 @@ import { CheckCircleIcon, ExclamationCircleIcon } from '@patternfly/react-icons'
 
 import type { CellProps } from './CellProps';
 
+type Phase = 'ConnectionFailed' | 'Ready' | 'Staging' | 'ValidationFailed';
+
+const statusIcons = {
+  ConnectionFailed: <ExclamationCircleIcon color="#C9190B" />,
+  Ready: <CheckCircleIcon color="#3E8635" />,
+  Staging: <Spinner size="sm" />,
+  ValidationFailed: <ExclamationCircleIcon color="#C9190B" />,
+};
+
+const phaseLabels = {
+  // t('Connection Failed')
```

Comment:
I think it is easier instead of adding a type, simply import t directly here and omit it from args